### DISCUSSION TITLE
Add new city request: Narva

### DISCUSSION THREAD

**Discussion by 62mkv** [EqualStreetNames/equalstreetnames#760] [New city]
Hi, kindly add this one to "the Hive": https://github.com/62mkv/equalstreetnames-narva

**Comment by jbelien** [comment]
Sure, you can transfer your repository to @EqualStreetNames organization and I'll take care of the last steps. 👍

**Reply by 62mkv** [reply]
in progress! thanks!

**Reply by 62mkv** [reply]
completed

**Comment by jbelien** [comment]
Work in progress: #774

**Reply by jbelien** [reply]
@62mkv Don't forget to add `MAPBOX_TOKEN` secret.

**Reply by jbelien** [reply]
There is some issue with the data update workflow everywhere, I'll fix that in the next few days.

**Reply by 62mkv** [reply]
`MAPBOX_TOKEN` secret is added

**Comment by jbelien** [comment]
🚀 Deployed: https://narva.equalstreetnames.eu/

**Comment by 62mkv** [comment]
hi @jbelien thank you so much and Happy New Year! 🥳 

May I ask: should some commit also update the nice view on the main page here? https://equalstreetnames.eu/ ? It does not display Narva in the box for Estonia for me :(

**Reply by 62mkv** [reply]
hmm, interesting, denmark/copenhagen is also not there 🤔 is there some criteria a city must comply with in order to be published on the main page?

**Reply by jbelien** [reply]
https://equalstreetnames.eu/ is completely independent of the EqualStreetNames project.
That website is managed by @RobinLinde, you should contact him.
I think it's indeed outdated because it refers to only 55 cities on the 63 we currently have.

**Reply by jbelien** [reply]
The official homepage of the EqualStreetNames project is https://equalstreetnames.org/

**Reply by RobinLinde** [reply]
Should be updated now. 2024 hasn't really been my year, so spent quite a lot less time programming (or mapping). Hoping that 2025 turns out a bit better.

**Reply by 62mkv** [reply]
oh cool! thanks @RobinLinde @jbelien !!
